src/main/java/de/tum/in/www1/artemis/repository/AchievementRepository.java

```diff
@@ -0,0 +1,14 @@
+package de.tum.in.www1.artemis.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import de.tum.in.www1.artemis.domain.Achievement;
+
+/**
+ * Spring Data JPA repository for the StudentExam entity.
```

Comment:
AchievementEntity

Reply:
Oh thanks, don´t know how I could overlook that 😄